### PR TITLE
УП-практикум_12.10.22г.

### PR DESCRIPTION
**Pull request by kokopeevski**
Калоян Кирилов Пеевски ( II група ) - факултетен номер: 2MI0600170

### CODE REVIEW

File: SI/Sem.02/Pract.02/Task9.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int firstNum, secondNum, thirdNum;
+	cout << "Enter three numbers: ";
+	cin >> firstNum >> secondNum >> thirdNum;
+	if (firstNum < secondNum && firstNum < thirdNum && secondNum < thirdNum)
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task9.cpp:12]: This solution doesn't work for the input 5 6 5 and there is a much simpler solution with just reordering(swapping) the numbers with three ifs

File: SI/Sem.02/Pract.02/Task10.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+			else
+			{
+				res = a / b;
+				break;
+			}
+		default:
+			cout << "Invalid operation" << endl;
+			break;
+	}
+	if (res != 0) cout << "Result: " << res << endl;
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task10.cpp:50]: This solution doesn't give output if your input is 5,+,-5 (the sum of 5 and -5 is zero)

File: SI/Sem.02/Pract.02/Task11.cpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int a, b, c;
+	cout << "Enter three numbers: ";
+	cin >> a >> b >> c;
+	int max, mid, min;
+	if (a>=b && a>=c)
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task11.cpp:13]: Your solution works but as similar to task 9 you can swap the numbers between them with three ifs and then print a*100 + b * 10 + c

File: SI/Sem.02/Pract.02/Task12.cpp [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+using namespace std;
+
+int main()
+{
+	int day, month;
+	cout << "Enter a date(day month): ";
+	cin >> day >> month;
+	cout << endl;
+	
+	if (month == 1 || month == 3 || month == 5 || month == 7 || month == 8 || month == 10 || month == 12)
```
Comment (GeorgiTerziev02) [inline SI/Sem.02/Pract.02/Task12.cpp:14]: The solution doesn't handle the case 31 12

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
There are some mistakes and missed edge cases but overally looks good.

File: SI/Sem.02/Pract.02/Task3.cpp [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int a, b;
+	cout << "Enter a two numbers: ";
+	cin >> a >> b;
+
+	(a > b) ? (cout << "The first one is bigger") : (cout << "The second one is bigger");
```
[the remark this answers is not in the document]
Comment (Justsvetoslavov) [reply SI/Sem.02/Pract.02/Task3.cpp:13]: Should always avoid repetition.

**Comment by Justsvetoslavov** [comment]
Review the new written PR rules and open new PRs if you would like (also check which ones are already merged ,as they will be duplicates).
[You can view them here](https://github.com/Justsvetoslavov/Introduction_to_programming)

**Pull request closed without merging**